Tasks/PythonScriptV0/task.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "6392F95F-7E76-4A18-B3C7-7F078D2F7700",
+    "name": "PythonScript",
+    "friendlyName": "Python Script",
+    "description": "Run a Python script.",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 135,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Run Python script",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "targetType",
+            "type": "radio",
+            "label": "Type",
+            "required": true,
+            "defaultValue": "filePath",
+            "helpMarkDown": "Target script type: File path or Inline",
+            "options": {
+                "filePath": "File path",
+                "inline": "Inline"
+            }
+        },
+        {
+            "name": "filePath",
+            "type": "filePath",
+            "label": "Script Path",
+            "visibleRule": "targetType = filePath",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "Path of the script to execute. Must be a fully qualified path or relative to $(System.DefaultWorkingDirectory)."
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "visibleRule": "targetType = inline",
+            "required": true,
+            "defaultValue": "",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "5000"
+            },
+            "helpMarkDown": "The Python script to run"
+        },
+        {
+            "name": "arguments",
+            "type": "string",
+            "label": "Arguments",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Arguments passed to the script execution, available through `sys.argv`."
+        },
+        {
+            "name": "pythonInterpreter",
+            "type": "string",
+            "label": "Python interpreter",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Absolute path to the Python interpreter to use. If not specified, the task will use the interpreter in PATH.",
```

Comment:
You could append:
"The `Use Python Version` task can be used to set the version of Python in PATH."

Reply:
How about I add this as a "tip" in the docs page?  That will keep the help text here brief.

Reply:
I like David's suggestion. Many users don't look at docs.

Reply:
![image](https://user-images.githubusercontent.com/33549821/40245037-7142af66-5a92-11e8-9a21-964a133d24b8.png)